druid/src/menu/mod.rs

```diff
@@ -0,0 +1,762 @@
+// Copyright 2019 The Druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! ## Menus
+//!
+//! Menus in `druid` follow a data-driven design similar to that of the main widget tree. The main
```

Comment:
Druid style guide: We say "Druid" when referring generally to the project, and "`druid`" only when we're talking explicitly about the crate, as in "Be sure to test both `druid` and `druid-shell` manually." This use is ambiguous, but informed by some use below I think we want 
"Druid".